src/Symfony/Component/HttpClient/CurlHttpClient.php

```diff
@@ -110,18 +108,18 @@ public function request(string $method, string $url, array $options = []): Respo
             ];
 
             if ('GET' === $method && !$options['body'] && $expectedHeaders === $pushedHeaders) {
-                $this->logger && $this->logger->info(sprintf('Connecting request to pushed response: %s %s', $method, $url));
+                $this->logger && $this->logger->debug(sprintf('Connecting request to pushed response: "%s" "%s"', $method, $url));
 
                 // Reinitialize the pushed response with request's options
                 $pushedResponse->__construct($this->multi, $url, $options, $this->logger);
 
                 return $pushedResponse;
             }
 
-            $this->logger && $this->logger->info(sprintf('Rejecting pushed response for "%s": authorization headers don\'t match the request', $url));
+            $this->logger && $this->logger->warning(sprintf('Rejecting pushed response for "%s": authorization headers don\'t match the request', $url));
```

Comment:
info or debug: this is not under control of the client - the remote should not be able to spam the logs

Reply:
warning does not spam the log either

Reply:
I could eventually move it to notice.